book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,487 @@
+It addresses both the accessibility requirement of the person who will be accompanied by a service animal, and also opens a line of communication with an 
+identified point of contact for anyone who may be impacted due to allergies or a fear of dogs. 
+This message was shared by email and on a company intranet - the two most common places to receive company communications, making it effective at reaching its staff and community.
+```
+
+> Dear Community,
+ 
+> Please be aware that a member of staff will be using a service dog when they attend the office from next week.
+> Please refrain from petting or interacting with the dog without the explicit consent of its owner. Many thanks for your cooperation.
+ 
+> If you have any allergies or require any adjustments, please contact the general facilities email address.
+
+
+Consider what channels your attendees with lived experience receive information from and try to match those in your event communications. 
+For example, some attendees may need to be called on the phone to receive information, or they may require a text instead of an email. 
+
+In many cases, you may need to develop a strategy for how to ensure that all attendees are able to receive event updates, and plan to use multiple different communication streams. 
+
+(cl-inclusive-events-comms-after-event)=
+## After the event 
+Finally, once your event has finished, you may want to consider how you will engage with the attendees. 
+
+This may involve sending out thank you's, a feedback survey, gift packs, or closing event information such as links and slides 
+for presentations, any summary notes, or other material that was delivered and able to be shared from the event. 
+
+Depending on the size and format of your event, consider how you will communicate the timeline for when attendees can expect any follow-up information. 
+It may take a significant portion of time to manually transcribe sessions and add captions before uploading to a video hosting platform so it is helpful for attendees to know to expect follow-up content.
+
+This can be especially important if your event involves people with lived experience giving feedback or input into a project. 
+It can be demoralizing for a person with lived experience to volunteer their time and expertise in a focus group or for a grant application, and then receive no further information. 
+Instead, consider setting a date when you or the research team will send an update, regardless of whether a potential project gets funded or not. 
+
+(cl-inclusive-events-comms-conclusion)=
+## Conclusion
+By demonstrating an open channel of communication about accessibility, and by being thorough in your communications plans, you will be well-positioned to support people with lived experience engaging in your event in a way that is safe and fulfilling. 
+
+(cl-inclusive-events-comms-additional-resources)=
+## Additional Resources
```

Comment:
Would it be good to aggregate all additional resources here (c.f. comments above on accessiblity guidelines etc)

Reply:
Added!